### ISSUE TITLE
Feature improvement: Scroll sync may need to align the first sections in both editor and preview

### ISSUE THREAD

**Issue by medifle** [HappenApps/Quiver#536]
Version 3.0.2 (81)

Current sync mechanism is easy to get lost:

![image](https://cloud.githubusercontent.com/assets/13296643/14063974/e63e9130-f41f-11e5-877a-1d1ede231bdf.png)

When I scroll to the immediate section:

![image](https://cloud.githubusercontent.com/assets/13296643/14063982/38aca826-f420-11e5-8a6c-c18e673cd048.png)

In [hackmd.io](https://hackmd.io), what I see is:

![image](https://cloud.githubusercontent.com/assets/13296643/14063940/348d350a-f41e-11e5-832c-f785430cbcd8.png)

Again, when I scroll to the immediate section:

![image](https://cloud.githubusercontent.com/assets/13296643/14063999/e2cee77e-f420-11e5-9aa0-58769fe3f7d3.png)

Not only [hackmd.io](https://hackmd.io), but [stackedit](stackedit.io) have done this scroll sync job very well. They are open source.

**Comment by ylian** [comment]
Yeah, I understand. Those other apps sync the positions of section headers. Quiver's scroll sync is quite basic at the moment, but will improve.
